src/docs/transactions.dox

```diff
@@ -149,26 +149,39 @@ the semantics supported by the drop configuration.
 
 Named snapshots are not durable: they do not survive WT_CONNECTION::close.
 
-@section transaction_timestamps Timestamps
+@section transaction_timestamps Application-specified Transaction Timestamps
 
-Applications can assign transactions explicit commit timestamps, then read
-as-of a timestamp.  Timestamps are communicated to WiredTiger as hexadecimal
-values with a length configurable at build time (default 8 bytes).
+Some applications have their own notion of time, including an expected commit
+order for transactions that may be inconsistent with the order assigned by
+WiredTiger.  We assume that applications can represent their notion of a
+timestamp as an integral value of some size that generally increases over
+time.  For example, a simple 64-bit integer could be incremented to generate
+transaction timestamps, if that is sufficient for the application.
+
+The application's timestamp size is specified as a number of bytes at build
+time, with <code>configure --with-timestamp-size=X</code>.  The default
+timestamp size is 8 bytes (i.e., 64 bits).  Setting a size of zero disables
+transaction timestamp functionality.
```

Comment:
Is there a performance reason to disable transaction timestamp functionality? (If so, maybe the default should be 0, or we should call that out so applications know to turn it off?)